php/libraries/LorisForm.class.inc

```diff
@@ -373,7 +373,7 @@ class LorisForm
                 $optionKey = (string) $optionKey;
             }
 
-            if ($optionKey === $this->getValue($el['name'])) {
+            if ($optionKey === $el['selected']) {
```

Comment:
I think you explained in the Loris meeting, but I can't remember.. why was this changed?